lib/domgen/ejb/generator.rb

```diff
@@ -86,11 +86,13 @@
                                 'cdi_types_test.java.erb',
                                 type + '/java/#{repository.ejb.qualified_cdi_types_test_name.gsub(".","/")}.java')
     end
-    g.template_set(:"ejb_#{type}_qa_aggregate") do |template_set|
```

Comment:
I am not sure why you changed this line but I would change it back unless you want to traul through our projects to see if it is needed ;-)

Reply:
Restored the deleted code.